server/config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We should make this file just an example file. And not version control the actual config file.